src/coreclr/src/debug/ee/debugger.cpp

```diff
@@ -9463,15 +9463,6 @@ void Debugger::SendExitAppDomainEvent(AppDomain* pRuntimeAppDomain)
 
     if (CORDebuggerAttached())
     {
-        if (pRuntimeAppDomain->IsDefaultDomain() )
-        {
-            // The Debugger expects to never get an unload event for the default Domain.
-            // Currently we should never get here because g_fProcessDetach will be true by
-            // the time this method is called.  However, we'd like to know if this ever changes
-            _ASSERTE(!"Trying to deliver notification of unload for default domain" );
```

Comment:
Isn't this the part we want to keep, since `pRuntimeAppDomain->IsDefaultDomain()` should always be true?

Reply:
ack - I read it backwards - I think you're right @elinor-fung

Reply:
This whole function is unreachable codepath. Delete it.